docs/sphinx/user_guide/profiling_with_caliper.rst

```diff
@@ -0,0 +1,179 @@
+
+  //Basic CPU using default build parameters
+  cmake ../
+
+  //With NVTX ON
+  cmake -DWITH_NVTX=ON ../
+
+  //With ROCTX ON
+  cmake -DWITH_ROCTX=ON ../
+
+Building RAJA with Caliper enabled requires pointing RAJA to the Caliper shared cmake file and enabling plugins::
+
+  cmake –DRAJA_ENABLE_RUNTIME_PLUGINS=ON –Dcaliper_DIR=${CALIPER_BUILD_DIR}/share/cmake/caliper ../
+
+As a quick build check we build the basic Caliper RAJA-Caliper example::
+
+  make raja-forall-caliper
+
+Finally, the Caliper annotated RAJA example may be executed via::
+
+  CALI_CONFIG=runtime-report ./bin/raja-forall-caliper
+
+If we build with NVTX enabled Caliper and CUDA enabled RAJA the end of the the run the program should output
+the following runtime information::
+
+  Path                     Time (E) Time (I) Time % (E) Time % (I)
+  C-version elapsed time   0.000820 0.000820   0.395169   0.395169
+  RAJA Seq daxpy Kernel    0.000655 0.000655   0.315502   0.315502
+  RAJA SIMD daxpy Kernel   0.000611 0.000611   0.294629   0.294629
+  RAJA OpenMP daxpy Kernel 0.013691 0.013691   6.598422   6.598422
+  RAJA CUDA daxpy Kernel   0.000118 0.000118   0.056827   0.056827
+
+The (E) column corresponds to exclusive timing; while the (I) column will correspond to inclusive timing.
+The left two columns are absolute time in seconds while the right two columns correspond to percentage of time
+within the program.
+
+.. note:: RAJA methods which offload to the GPU will require a synchronous GPU policy to ensure that the kernel
+          has completed prior to the total runtime being reported. For asynchronous kernels Caliper offers the
+          `hip.gputime` or `cuda.gputime` service which may be added to CALI_CONFIG to accurately capture kernel
+          time.
+
+========================================
+Profiling RAJA kernels via kernel naming
+========================================
+Caliper annotations of RAJA kernels work through the RAJA kernel naming mechanism which is currently only supported in forall
+and launch. The ``RAJA::Name`` container holds a string and used for profiling in Caliper. Kernels
+which are not provided a name are ommited from Caliper profiling::
+
+    RAJA::forall<RAJA::seq_exec>(RAJA::RangeSegment(0, N),
+    RAJA::Name("RAJA Seq daxpy Kernel"), [=] (int i) {
+
+        a[i] += b[i] * c;
+
+  });
+
+.. note:: The RAJA Name feature lives under the RAJA namespace.
+
+.. note:: The RAJA Name may be placed anywhere in the RAJA reducer parameter list
+
+.. note:: When RAJA is not configured with Caliper, NVTX and ROCTX profiling can
+          still be configured with RAJA using the following CMAKE variables respectively::
+            -DRAJA_ENABLE_NV_TOOLS_EXT=ON
+            -DRAJA_ENABLE_ROCTX=ON
+         When Caliper is configured with RAJA all performance profiling is performed by Caliper.
```

Comment:
Is this last sentence also supposed to be a note? If so, it is missing the note annotation. If not, it should be separated from the previous note by a blank line and not indented.